plugins/file_chooser/linux/src/file_chooser_plugin.cc

```diff
@@ -86,19 +86,21 @@ static void ProcessAttributes(const Json::Value &method_args,
 // string, then this returns a file saver dialog.
 //
 // If the method is not recognized as one of those above, will return a nullptr.
-static GtkFileChooserNative *CreateFileChooserFromMethod(
+static GtkFileChooserDialog *CreateFileChooserFromMethod(
     const std::string &method, const std::string &ok_button) {
-  GtkFileChooserNative *chooser = nullptr;
+  GtkFileChooserDialog *chooser = nullptr;
```

Comment:
nit: Perhaps we can change this to `GtkWidget*` and get rid of the casting, then return `GTK_DIALOG(chooser)` at the end of the function?  I'm okay with this implementation either way.

Reply:
This is really a good suggestion, I will modify this function as you said.